dominion/deck.py

```diff
@@ -20,14 +20,14 @@ class Deck:
     coins: int
     game: Game
 
-    def __init__(self, game: Game):
+    def __init__(self, game: Game, draw_pile: t.List[CardTypes] = None):
```

Comment:
This initial draw pile is always the same and taking a parameter is not allowed by the dominion rules.
Thank you!